### PR TITLE
Module event type

### PR DESCRIPTION
**Pull request by tomhuynh**

### CODE REVIEW

File: src/app/module/module-entities-add/module-entities-add.component.ts [hunk 1]
```diff
@@ -35,9 +35,21 @@ export class ModuleEntitiesAddComponent implements OnInit {
...
-      this.fields = Fields;
-    }, err => { });
+    this.moduleService.getModule(id).subscribe(moduleInfo => {
+      this.moduleService.getEntitiesByModuleId(id, isOptionRequired).subscribe((Fields) => {
+        if (moduleInfo.entityType === 'Default') {
+          this.fields = Fields;
+        } else if (moduleInfo.entityType === 'Article') {
+          const localFields: Field[] = [];
+          Fields.map((value, index) => {
+            if (value.name === 'newstitle' || value.name === 'newsLink' || value.name === 'news' || value.name === 'newsDate') {
```
Comment (heathbobby) [inline src/app/module/module-entities-add/module-entities-add.component.ts]: I thought we were going to change this.

File: src/app/module/module-entities-edit/module-entities-edit.component.ts [hunk 2]
```diff
@@ -39,18 +39,32 @@ export class ModuleEntitiesEditComponent implements OnInit {
...
-          value.fieldEntityValue.fieldValue = checkBoxFieldsValue.split(',');
+    this.moduleService.getModule(id).subscribe(moduleInfo => {
+      this.moduleService.getEntitiesValuesByEntityId(id).subscribe((enityFieldValue) => {
+        this.fieldValue = enityFieldValue;
+        if (moduleInfo.entityType === 'Default') {
+          this.fieldValue = enityFieldValue;
+        } else if (moduleInfo.entityType === 'Article') {
+          const localFields: Field[] = [];
+          enityFieldValue.map((value, index) => {
+            if (value.name === 'newstitle' || value.name === 'newsLink' || value.name === 'news' || value.name === 'newsDate') {
```
Comment (heathbobby) [inline src/app/module/module-entities-edit/module-entities-edit.component.ts]: These are wrong.  They don't represent the values we talked about.

File: src/app/module/module-entities-add/module-entities-add.component.ts (hunk 1 above)
Comment (tomhuynh) [reply src/app/module/module-entities-add/module-entities-add.component.ts]: I did the logic in Retrieve and forgot doing same thing in Add/Edit page. Fixed and pushed.

File: src/app/module/module-entities-edit/module-entities-edit.component.ts (hunk 2 above)
Comment (tomhuynh) [reply src/app/module/module-entities-edit/module-entities-edit.component.ts]: I did the logic in Retrieve and forgot doing same thing in Add/Edit page. Fixed and pushed.

**Pull request merged**